frontend/src/components/Window.js

```diff
@@ -12,6 +12,7 @@ import Section from './window/Section';
 import Dropzone from './Dropzone';
 import { INITIALLY_CLOSED } from '../constants/Constants';
 
+const EMPTY_OBJECT = {};
```

Comment:
We've already talked about this during a frontend meeting with Teo. We want to be sure what is used and that it wasn't altered in any way bo some other component, since it'll be a reference to a js object.